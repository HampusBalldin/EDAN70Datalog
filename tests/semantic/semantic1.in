TYPEOF('A, [String String]).
TYPEOF('B, [String]).
A(x, y) :- B(x), B(y).
A(x, y, z) :- B(x, y, z).
