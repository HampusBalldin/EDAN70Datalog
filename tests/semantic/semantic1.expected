Different Arities for predicate symbol: A
Different Arities for predicate symbol: B
